pydy/tests/test_models.py

```diff
@@ -19,8 +16,8 @@ def test_multi_mass_spring_damper():
 
     sys = multi_mass_spring_damper()
 
-    assert Counter(sys.constants_symbols) == Counter([k0, c0, m0])
-    assert sys.specifieds_symbols == list()
+    assert sys.constants_symbols == {k0, c0, m0}
```

Comment:
I can see how a set may make more sense for these, but I don't think it should be changed unless you want to properly deprecate it.